### ISSUE TITLE
System.FormatException in Guild Library

### ISSUE THREAD

**Issue by sguionni** [Dimencia/LuteBot3#23]
Hello,

when i click on the Play button after choosing a track in the Guild Library, i have this exception (note that this happen only the first time, all is working after):

```
System.FormatException: Le format de la chaîne d'entrée est incorrect.
   à System.Number.ParseSingle(String value, NumberStyles options, NumberFormatInfo numfmt)
   à SimpleML.NeuralNetwork.Load(String path)
   à LuteBot.TrackSelection.TrackSelectionManager.GetFlutePrediction()
   à LuteBot.TrackSelection.TrackSelectionManager.LoadTracks(Dictionary`2 channels, Dictionary`2 tracks)
   à LuteBot.LuteBotForm.PlayerLoadCompleted(Object sender, AsyncCompletedEventArgs e)
   à LuteBot.Core.Midi.MidiPlayer.HandleLoadCompleted(Object sender, AsyncCompletedEventArgs e)
   à Sanford.Multimedia.Midi.Sequence.OnLoadCompleted(Object sender, RunWorkerCompletedEventArgs e)
   à System.ComponentModel.BackgroundWorker.OnRunWorkerCompleted(RunWorkerCompletedEventArgs e)
```

**Comment by dquartino** [comment]
Hi !
This error is due to an invalid region number formatting. French decimal decimal use comma instead of dot. To fix this error you need to change the decimal  separator in windows : [check here](https://answers.microsoft.com/en-us/msoffice/forum/all/changing-decimal-separator-from-comma-to-dot/953556fb-00d0-45ad-bd4f-6ca8afa5025a)

**Comment by Dimencia** [comment]
> Hi ! This error is due to an invalid region number formatting. French decimal decimal use comma instead of dot. To fix this error you need to change the decimal separator in windows : [check here](https://www.youtube.com/watch?v=RwJvjH3KXVg)

Perfect, thanks.  I just pushed a hotfix that should fix it, will test it out myself later and close this with any luck

**Comment by Dimencia** [comment]
> This indroduce that error:
> 
> ```
> System.IO.DirectoryNotFoundException: Impossible de trouver une partie du chemin d'accès 'C:\WINDOWS\system32\lib\channelNeural'.
>    à System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
>    à System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
>    à System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
>    à System.IO.StreamReader..ctor(String path, Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize, Boolean checkHost)
>    à System.IO.StreamReader..ctor(String path)
>    à SimpleML.NeuralNetwork.Load(String path)
>    à LuteBot.TrackSelection.TrackSelectionManager.GetFlutePrediction()
>    à LuteBot.TrackSelection.TrackSelectionManager.LoadTracks(Dictionary`2 channels, Dictionary`2 tracks)
>    à LuteBot.LuteBotForm.PlayerLoadCompleted(Object sender, AsyncCompletedEventArgs e)
>    à LuteBot.Core.Midi.MidiPlayer.HandleLoadCompleted(Object sender, AsyncCompletedEventArgs e)
>    à Sanford.Multimedia.Midi.Sequence.OnLoadCompleted(Object sender, RunWorkerCompletedEventArgs e)
>    à System.ComponentModel.BackgroundWorker.OnRunWorkerCompleted(RunWorkerCompletedEventArgs e)
> ```

Should be fixed in 3.5.2, will leave the issue open a while in case anyone has further issues

"Voice mode" is lutemod, not lutebot; lutemod has no key configuration
